addon/addon/resolvers/classic/index.js

```diff
@@ -130,22 +128,22 @@ function resolveOther(parsedName) {
   }
 }
 
-const Resolver = EmberObject.extend({
-  resolveOther,
-  parseName,
-  pluralizedTypes: null,
-  moduleRegistry: null,
+class Resolver extends EmberObject {
+  resolveOther = resolveOther;
+  parseName = parseName;
+  pluralizedTypes = null;
+  moduleRegistry = null;
```

Comment:
No need to do either of these as class fields. they are always assigned directly in the constructor